blackjax/mcmc/sgld.py

```diff
@@ -0,0 +1,33 @@
+"""Public API for the Stochastic gradient Langevin Dynamics kernel."""
+from typing import Callable, NamedTuple
+
+from blackjax.mcmc.diffusion import overdamped_langevin
+from blackjax.types import PRNGKey, PyTree
+
+__all__ = ["SgLDState", "SgLDInfo", "init", "kernel"]
+
+
+class SgLDState(NamedTuple):
```

Comment:
nitpick: the lowercase g looks odd to me. The standard way of writing the abreviation (in papers) is "SGLD", so `SGLDState` would feel more natural to me